docs/03_server/10_integration/06_notify/03_symphony.md

```diff
@@ -14,51 +14,77 @@ tags:
 To make Symphony services available to Genesis, you need to provision [symphony service](https://symphony.com/participate) and configure a [symphony bot](https://docs.developers.symphony.com/developer-tools/developer-tools/bdk-2.0).
 
 ## Importing Symphony
-Symphony is a separate module of the Genesis low-code platform. This gives platform users a better way to manage third-party dependencies, as the Symphony BDK has a number of transitive dependencies.
+Symphony is a separate module of the Genesis low-code platform. This gives you a better way to manage third-party dependencies, as the Symphony BDK has a number of transitive dependencies.
 
 ### Adding the Symphony module data schema
-In order to add the Symphony route tables to your application, add the following in your application dictionary cache module: 
+In order to add the Symphony route tables to your application, add the following line to the **build.gradle.kts** file in your application's **dictionary-cache** folder: 
 
-```kotlin
-api("global.genesis:genesis-symphony-config")
+```kotlin {3}
+dependencies {
+  ...
+  api("global.genesis:genesis-symphony-config")
+  ...
+}
 ```
 
 ### Adding the Symphony module to deployment
-In order to add the Symphony distribution to your application deployment, add the following in your application deployment module: 
+To add the Symphony route tables to your application deployment, add the following line to the **build.gradle.kts** file in your application's **deploy** folder:
 
 ```kotlin
-genesisServer(
+dependencies {
+  ...
+  genesisServer(
     group = "global.genesis",
     name = "genesis-symphony-distribution",
     version = properties["symphonyVersion"].toString(),
     classifier = "bin",
     ext = "zip"
   )
+  ...
+}
 ```
 
-### Adding the Symphony module endpoints to GENESIS_NOTIFY
-In order to add the Symphony endpoints to your GENESIS_NOTIFY process, add the following package in your process definition: 
-```
-global.genesis.symphony.gateway.endpoints
-```
-and the following jar to the GENESIS_NOTIFY_PROCESS classpath
+## Running locally
+To run locally, you need to copy your **genesis-notify-processes.xml** file. If you are using the IntelliJ plugin, you can find this in the folder **.genesis-home/genesis-notify/cfg/**.
+
+Paste this file into the **site-specific** folder at **server/jvm/alpha-site-specific/src/main/resources/cfg/**.
```

Comment:
Should we use alpha specifically outside of the training

Reply:
No, I should have changed that and I shall shortly